lib/EnsEMBL/REST/Controller/Info.pm

```diff
@@ -331,6 +331,131 @@ sub consequence_types_GET {
   return;
 }
 
+### The following methods were ported from EG REST...
+
+sub ensgen_version : Chained('/') PathPart('info/eg_version') :
+  ActionClass('REST') : Args(0) { }
+
+sub ensgen_version_GET {
+  my ( $self, $c ) = @_;
+  $c->log()->info("Retrieving EG version from registry");
+  # lazy load the registry
+  $c->model('Registry')->_registry();
+  my $lookup = $c->model('Registry')->_lookup();
+  $self->status_ok( $c, entity => { version => $lookup->adaptor->data_release->ensembl_genomes_version } );
+  return;
+}
+
+sub genomes_name : Chained('/') PathPart('info/genomes') :
+  ActionClass('REST') : Args(1) { }
```

Comment:
It's a bit odd to have a mandatory argument but then ignore it and delegate to a follow on method.

Reply:
Not planning to change this, unless you feel strongly and can suggest better option?

Reply:
Fair enough... I'm not prepared to declare a one true way right now. Catalyst is too finicky around these decorators.